client/state/login/actions.js

```diff
@@ -1,41 +1,70 @@
 /**
+ * External dependencies
+ */
+import { translate } from 'i18n-calypso';
+
+/**
  * Internal dependencies
  */
+import config from 'config';
+import request from 'superagent';
```

Comment:
Nit: superagent and lodash are external dependencies. It would be nice to drop that distinction at some point ...